### DISCUSSION TITLE
Chromebook Rick-covery Shim?

### DISCUSSION THREAD

**Discussion by FunnyCorgi** [3kh0/ext-remover#1131] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
I would like to in general learn how to mod RMA shims, but for my first one I want make a "Rick-covery shim" that rickrolls whoever plugs it in. 

Mostly this is just an idea I had for getting into modding RMA shims.

**Comment by S-PScripts** [comment]
no
